src/main/java/seedu/address/model/task/ModuleIsDonePredicate.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.model.task;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.StringUtil;
+
+/**
+ * Tests that a {@code Task}'s {@code isDone} is false.
+ */
+public class ModuleIsDonePredicate implements Predicate<Task> {
+
+    private final List<String> isDone;
+
+    public ModuleIsDonePredicate(List<String> isDone) {
+        this.isDone = isDone;
+    }
+
+    @Override
+    public boolean test(Task task) {
+        return isDone.stream()
+                .anyMatch(isDone -> StringUtil.containsWordIgnoreCase(String.valueOf(task.isDone()), isDone));
```

Comment:
Would `return !task.isDone` be better?

Feels like this is overcomplicating it